fetch.bs

```diff
@@ -2354,6 +2354,49 @@ X-Content-Type-Options           = "nosniff" ; case-insensitive</pre>
 pertain to them. Also, considering "<code>image</code>" was not compatible with deployed content.
 
 
+<h3 id=cross-origin-read-blocking>Cross-Origin Read Blocking (CORB)</h3>
+
+<h4 lt="should response to request be blocked due to corb" dfn id=should-response-to-request-be-blocked-due-to-corb?>Should
+<var>response</var> to <var>request</var> be blocked due to CORB?</h4>
```

Comment:
I think rather than a heading we should introduce this algorithm with a paragraph. I'm not a big fan of using headings.

Reply:
We should also try to avoid the question mark in the ID. That was not really intentional with the other algorithm in Fetch and complicates linking towards it.

Reply:
Done:
- I've more-or-less copied the first 2 sentences from the [CORB explainer](https://chromium.googlesource.com/chromium/src/+/master/services/network/cross_origin_read_blocking_explainer.md#What-kinds-of-requests-are-CORB_eligible) into an introductory paragraph in the "Cross-Origin Read Blocking" section being added in this PR.
- I've also got rid of a separate id attribute (which means there is no question mark at the end of the link)